escuro_johndeniel/Countries/script.js

```diff
@@ -0,0 +1,33 @@
+document.getElementById("search-button").addEventListener("click", function () {
+  var country = document.getElementById("country-input").value;
```

Comment:
Always use "let" keyword in declaring variables over the "var" keyword.

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g25d183194fb_0_0